sjb/config/common/test_cases/origin_installed_release.yml

```diff
@@ -87,6 +87,7 @@ extensions:
                          -e etcd_data_dir="${ETCD_DATA_DIR}" \
                          -e openshift_master_default_subdomain="${local_ip}.nip.io"             \
                          -e openshift_pkg_version="$( cat ./ORIGIN_PKG_VERSION )"               \
+                         -e openshift_image_tag="$( cat ./ORIGIN_COMMIT )"                      \
```

Comment:
Should this be the tag not the commit?

Reply:
In this case we want to use the images that got built by the job.  Once every job is building images the same way, this value would be set to whatever the tag is for the combined job.